envoy.dependency.cve_scan/tests/test_integration.py

```diff
@@ -0,0 +1,197 @@
+
```

Comment:
@htuch ive added integration tests here

it adds quite a lot of code for the test data, but otoh it provides a reference implementation which is quite helpful

working with the nist data is a pita as its huge and unwieldy, but ill fish some more CVEs out of it and add some further tests

there are probably some cleanups also, but would like to "move to close" on this PR